docs/framework/get-started/overview.md

```diff
@@ -20,85 +20,90 @@ author: "rpetrusha"
+
+The .NET Framework can be hosted by unmanaged components that load the common language runtime into their processes and initiate the execution of managed code, thereby creating a software environment that exploits both managed and unmanaged features. The .NET Framework not only provides several runtime hosts but also supports the development of third-party runtime hosts.
+
+For example, ASP.NET hosts the runtime to provide a scalable, server-side environment for managed code. ASP.NET works directly with the runtime to enable ASP.NET apps and XML Web services, both of which are discussed later in this topic.
+
+Internet Explorer is an example of an unmanaged app that hosts the runtime (in the form of a MIME type extension). Using Internet Explorer to host the runtime enables you to embed managed components or Windows Forms controls in HTML documents. Hosting the runtime in this way makes managed mobile code possible, but with significant improvements that only managed code offers, such as semi-trusted execution and isolated file storage.
+
+The following illustration shows the relationship of the common language runtime and the class library to your apps and to the overall system. The illustration also shows how managed code operates within a larger architecture.
+
+![Managed code within a larger architecture](../../../docs/framework/get-started/media/circle.gif "circle")
+.NET Framework in context
+
+The following sections describe the main features of the .NET Framework in greater detail.
+
+## Features of the common language runtime
+
+The common language runtime manages memory, thread execution, code execution, code safety verification, compilation, and other system services. These features are intrinsic to the managed code that runs on the common language runtime.
+
+Regarding security, managed components are awarded varying degrees of trust, depending on a number of factors that include their origin (such as the Internet, enterprise network, or local computer). This means that a managed component might or might not be able to perform file-access operations, registry-access operations, or other sensitive functions, even if it's used in the same active app.
+
+The runtime enforces code access security. For example, users can trust that an executable embedded in a webpage shows an animation on screen or plays a song but cannot access their personal data, file system, or network. The security features of the runtime thus enable legitimate exceptionally feature rich Internet-deployed software.
```

Comment:
Idk what "embed" means in this context. I'm more familiar with the concept of "executing" in this context. I defer to @mairaw @BillWagner @rpetrusha: Kill this paragraph? ... or provide a different example of CAS here?